test/NuGet.Core.Tests/NuGet.Build.Tasks.Console.Test/MSBuildStaticGraphRestoreTests.cs

```diff
@@ -869,6 +871,7 @@ public void GetTargetFrameworkInfos_WithCustomAliases_InfersCorrectTargetFramewo
                         { "TargetFrameworkMoniker", $"{FrameworkConstants.FrameworkIdentifiers.Net},Version=4.6.1" },
                         { "TargetPlatformIdentifier", "" },
                         { "TargetPlatformVersion", "" },
+                        { "TargetPlatformVersion", "" },
```

Comment:
I think you meant Moniker?
```suggestion
                        { "TargetPlatformMoniker", "" },
```

Reply:
yep...fixed that in a future commit. The change had stayed in my local changes.